packages/plugin/src/generators/create-package/create-package.ts

```diff
@@ -39,9 +39,6 @@ export async function createPackageGenerator(
   tasks.push(installTask);
 
   await createCliPackage(host, options, pluginPackageName);
-  if (options.e2eTestRunner !== 'none') {
-    tasks.push(await addE2eProject(host, options));
-  }
```

Comment:
Are we just never creating this e2e project?

Reply:
We're waiting until we have a better solution.